zzzz_modular_occulus/code/modules/long_range_scanner/long_range_scanner.dm

```diff
@@ -3,7 +3,7 @@
 
 /obj/effect/overmap/ship/Process()
 	if(scanners)//do we have any scanners in the list?
-		var/x = scanners.len
+		//var/x = scanners.len
```

Comment:
This can just be removed, it was used to loop through the scanners originally (though that's been switched to a `for(x in y)` loop)

Reply:
Easy enough to do. Gimme a moment

Reply:
IIRC I intended to remove it after testing that removing the var would stop compile warnings... then forgot to actually do it.